darshan-util/darshan-convert.c

```diff
@@ -30,7 +30,10 @@ int usage (char *exename)
     fprintf(stderr, "       Converts darshan log from infile to outfile.\n");
     fprintf(stderr, "       rewrites the log file into the newest format.\n");
     fprintf(stderr, "       --bzip2 Use bzip2 compression instead of zlib.\n");
-    fprintf(stderr, "       --obfuscate Obfuscate items in the log.\n");
```

Comment:
Would it be worth keeping --obfuscate for compatibility?  (setting it would implicitly set each of the individual flags, so the comment/usage in that case would be something like "Enable all obfuscate options (jobid, uid, exe, and names)"

Reply:
good call, i fixed this by adding back the original option that sets all flags